app/views/action_page/index.html.erb

```diff
@@ -10,6 +10,11 @@
       <div class="row">
         <% array_of_3_actions.each do |actionPage|%>
             <div>
+              <h5>
```

Comment:
This new version will result in an empty `h5` if `category` isn't present. It would be better to put the tag inside the conditional, like
```
<% if actionPage.category.present? %>
  <h5><%= actionPage.category.title %></h5>
<% end %>
```